src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/metadata/ProjectSampleMetadataAjaxController.java

```diff
@@ -49,7 +50,7 @@ public ProjectSampleMetadataAjaxController(UIMetadataImportService metadataImpor
 	@PostMapping("/upload/file")
 	@ResponseBody
```

Comment:
All your URI's have `/upload` move that up into the controller mapping

Reply:
Please see 134fb930ef25597c0aa64fce0eb528bb82d414fe.